rpcd/etc/openstack_deploy/user_apt_packages.yml

```diff
@@ -0,0 +1,11 @@
+# This file will contain any apt packages that the deployer would like to pin to a particular version.
```

Comment:
I don't see the justification for breaking this out into a separate file. Unless this is part of some more generalised approach to separate user_extra_variables into multiple files it seems a bit random just to do it for this one var.